.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "wine"]
 	path = wine
-	url = ../wine
+	url = https://github.com/ValveSoftware/wine
```

Comment:
The reason this needs to be an absolute path is for forks so that the fork doesn't also need a fork of wine.

Reply:
I suppose the origin path could also be set in CI so this isn't needed.